raiden_contracts/contracts/MonitoringService.sol

```diff
@@ -43,15 +43,15 @@ contract MonitoringService is Utils {
      *  Events
      */
 
-    event NewDeposit(address receiver, uint amount);
+    event NewDeposit(address indexed receiver, uint amount);
     event NewBalanceProofReceived(
         uint256 reward_amount,
         uint256 nonce,
```

Comment:
The MS might want to specifically filter for events with `>=nonce` - to check if the balance proof has already been updated. So consider adding `indexed` also here.